src/redux/flows/flows.reducer.ts

```diff
@@ -159,6 +164,10 @@ function _deleteFlow(state: FlowState, flowId: string): FlowState {
   return stateWithoutFlow;
 }
 
+function _importFlow(state: FlowState, customYAML: string): FlowState {
+  return _createNewFlow(state, customYAML);
+}
+
 function _createNewFlow(state: FlowState, customYAML?: string): FlowState {
```

Comment:
Could you write some tests for `_createNewFlow`? I think this will help when we update this in the future.